extensions/web-tracker/javascript/src/analytics.js-integration-apache-unomi.js

```diff
@@ -19,23 +19,26 @@
 var integration = require('@segment/analytics.js-integration');
 var extend  = require('extend');
 
-var Unomi = module.exports = integration('Apache Unomi')
+var Unomi = (module.exports = integration('Apache Unomi')
```

Comment:
```suggestion
var Unomi = module.exports = integration('Apache Unomi')
```

Reply:
This was done this way because all the other analytics.js integrations use this format.

Reply:
I don't see any reason to add parentheses here.

We assume that js var is passed as a reference, and 
```
var x = y = "toto"
// x will be "toto"
// y will be "toto"
```

Reply:
Not sure why this was done, but probably for legibility. Again here we are re-using the patterns in the analytics.js source code.

Reply:
Here are examples:
https://github.com/segmentio/analytics.js-integrations/blob/master/integrations/adobe-analytics/lib/index.js
https://github.com/segmentio/analytics.js-integrations/blob/master/integrations/facebook-pixel/lib/index.js
https://github.com/segmentio/analytics.js-integrations/blob/master/integrations/adobe-target/lib/index.js